boards/saml21-xpro/include/periph_conf.h

```diff
@@ -118,6 +118,32 @@ static const spi_conf_t spi_config[] = {
 #define RTT_NUMOF           (1)
 /** @} */
 
+/**
+ * @name ADC Configuration
+ * @{
+ */
+#define ADC_NUMOF                          ADC_0_CHANNELS
+#define ADC_0_EN                           1
+#define ADC_MAX_CHANNELS                   14
+/* ADC 0 device configuration */
+#define ADC_0_DEV                          ADC
```

Comment:
Same comment as [here](https://github.com/RIOT-OS/RIOT/pull/6666#discussion_r115228560). It would be great to only have the `adc_channel_t` initialized and ADC_NUMOF defined in periph_conf.h.